packages/malloy/src/lang/test/parse-expects.ts

```diff
@@ -240,6 +240,11 @@ function eToStr(e: Expr, symbols: ESymbols): string {
       return e.node;
     case 'regexpMatch':
       return `{${subExpr(e.kids.expr)} regex-match ${subExpr(e.kids.regex)}}`;
+    case 'in': {
+      return `{${subExpr(e.kids.e)} ${e.not ? 'not in' : 'in'} ${e.kids.oneOf
+        .map(o => `{${subExpr(o)}}`)
```

Comment:
I might put the `{}` around the whole list rather than each element? Just for consistency with the other formatters here, normally single values don't get `{}` around them: `x in (1, 2, 3)` would then be `{x in {1,2,3}}` rather than `{x in {1},{2},{3}}`. And maybe `.join(', ')` 😊 

Not that it matters...

Reply:
ok except maybe not comma space